cli/Makefile

```diff
@@ -10,22 +10,17 @@ assets:
 	go-bindata --pkg assets -o assets/assets.go assets/**/*.html
 
 HAS_GOX := $(shell command -v gox;)
-HAS_DEP := $(shell command -v dep;)
 HAS_BINDATA := $(shell command -v go-bindata;)
 
 .PHONY: deps
 # get dependencies
 deps:
-ifndef HAS_DEP
-	go get -u github.com/golang/dep/cmd/dep
-endif
 ifndef HAS_GOX
 	go get github.com/mitchellh/gox
 endif
 ifndef HAS_BINDATA
 	go get github.com/hasura/go-bindata/go-bindata
 endif
```

Comment:
Let's add `go mod download` as a step to pre-download packages under `make deps`